packages/@aws-cdk/aws-stepfunctions-tasks/README.md

```diff
@@ -37,6 +37,8 @@ This module is part of the [AWS Cloud Development Kit](https://github.com/aws/aw
   - [UpdateItem](#updateitem)
 - [ECS](#ecs)
```

Comment:
I'm not super familiar with ECS. Should you get @MrArnoldPalmer who owns ECS to take a look and see if it matches up correctly and the right interfaces are used?

Reply:
added him to review this.